### DISCUSSION TITLE
Decoding number to text

### DISCUSSION THREAD

**Discussion by zanjabil2502** [openai/whisper#261] [Q&A] [answered]
hi, i want to ask about decoding whisper, whisper will predict the text of a number to be a number, how to convert a number to a text, for example 1 becomes "one"?
This also happens when I use a language other than English.

**Comment by ArtyomZemlyak** [comment]
Hi!
You can try apply postprocessing on text. For example using this library:
https://pypi.org/project/num2words/

**Comment by jongwook** [comment, marked as answer]
Please see:

- https://github.com/openai/whisper/discussions/1041#discussioncomment-5224837
